app/src/main/java/com/sdp/movemeet/backend/firebase/storage/StorageImageManager.java

```diff
@@ -0,0 +1,49 @@
+package com.sdp.movemeet.backend.firebase.storage;
+
+import android.net.Uri;
+
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.storage.StorageReference;
+import com.google.firebase.storage.StorageTask;
+import com.sdp.movemeet.backend.BackendManager;
+import com.sdp.movemeet.backend.providers.BackendInstanceProvider;
+import com.sdp.movemeet.models.Image;
+
+public class StorageImageManager implements BackendManager<Image> {
+
+    private final StorageReference storageReference;
+
+    public StorageImageManager() {
+        this.storageReference = BackendInstanceProvider.getStorageInstance().getReference();
+    }
+
+    @Override
+    public StorageTask add(Image image, String path) {
+        return storageReference.child(path).putFile(image.getImageUri());
+    }
+
+    @Override
+    public Task<?> set(Image object, String path) {
```

Comment:
I think it could be a good idea to implement those methods as well, although there are not used (yet). Otherwise, you can change "return null" by "throw new UnsupportedOperationException()".

Reply:
You're right Victor! Kepler made the same comment! For the moment I modified this file with some `throw new UnsupportedOperationException()` for the methods we don't use! I guess we don't need those methods for the Image objects for the moment, but if we need them in the future, we will implement them and remove the `throw new UnsupportedOperationException()` parts 👍.